common/src/main/java/de/florianmichael/rclasses/common/memory/InvalidAlignmentException.java

```diff
@@ -0,0 +1,30 @@
+/*
+ * This file is part of RClasses - https://github.com/FlorianMichael/RClasses
+ * Copyright (C) 2023 FlorianMichael/EnZaXD and contributors
+ *
+ *  Licensed under the Apache License, Version 2.0 (the "License");
+ *  you may not use this file except in compliance with the License.
+ *  You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package de.florianmichael.rclasses.common.memory;
+
+import javax.management.InvalidApplicationException;
+
+/***
+ * An exception that is fired when the memory stack is used with an invalid alignment
+ */
+public class InvalidAlignmentException extends InvalidApplicationException  {
+
+    public InvalidAlignmentException(long alignment) {
+        super(String.format("Invalid alignment: %d", alignment));
```

Comment:
Why use String.format? I don't see the reason why we would need that here, you can just do:
"Invalid alignment: " + alignment

Reply:
I always use `String.format`, but in this case, just using concat operator will be faster, good point.